### PR TITLE
Eslint - Add no-constant-condition rule

### PR DESCRIPTION
**Pull request by San4es**
This rule is a part of the "eslint:recommended" configuration.
Info: https://eslint.org/docs/rules/no-constant-condition

### CODE REVIEW

File: js/exporter/excel_creator.js [hunk 1]
```diff
@@ -69,16 +70,15 @@ var ExcelCreator = Class.inherit({
             this._maxIndex[1] = Number(cellIndex);
         }
 
-        while(true) {
```
Comment (San4es) [inline js/exporter/excel_creator.js:72]: @jsdmitry @IgnatovDan @nightskylark 
Or just replace `while(true)` with `for(;;)` here without additional changes?🤨

Comment (IgnatovDan) [reply js/exporter/excel_creator.js:72]: Change just to fit lint rules... Something like this?

        var isCellIndexFound = false;
        while(!isCellIndexFound) {
            charCode = 65 + ((cellIndex >= max) ? (cellIndex % max) : Math.ceil(cellIndex));
            sheetIndex = String.fromCharCode(charCode) + sheetIndex;
 
            if(cellIndex >= max) {
                cellIndex = Math.floor(cellIndex / max) - 1;
            } else {
                isCellIndexFound = true;
            }
        }

**Review by IgnatovDan — APPROVED** [review]

File: js/exporter/excel_creator.js (hunk 1 above)
Comment (San4es) [reply js/exporter/excel_creator.js:72]: Thank you @IgnatovDan! Done.

**Comment by San4es** [comment]
Another option is to use `checkLoops` that allows constant expressions in loops:

    no-constant-condition: ["error", { "checkLoops": false }]

Your thoughts?

**Review by nightskylark — APPROVED** [review]

**Review by jsdmitry — APPROVED** [review]

**Pull request merged**